resources_pages/quiz.md

```diff
@@ -5,81 +5,95 @@ title: MDS quizzes
 
 
 ### Overview
+
 With the exception of [project courses](https://ubc-mds.github.io/2019-08-22-project-courses/), MDS courses will have two quizzes throughout the block, typically one in the middle of the course and another after the course ends during the free days between blocks. The quizzes for a course will appear on the [MDS calendar](https://ubc-mds.github.io/calendar/) (ideally before the course starts) so that students are aware of the timing. We anticipate most quizzes being 45 minutes long.
 
 ### Purpose
+
 The purpose of quizzes are to be a "sanity check" that students are keeping up with the essential material. While labs may go more in-depth into a topic, the quizzes cover essential material that we expect students to have mastered and to retain.
 
 ### Difficulty
+
 In light of the purpose above, quizzes are intended to be less difficult than the labs. They are not meant to trick you nor to contain unexpected material. In short, if you did the labs without an unreasonable amount of collaboration with your classmates, you should succeed on the quizzes.
 
+### Quiz Format and Platform
+
+This year, quizzes will be held on PrairieLearn. Instead of quizzes being held at the same time for every student, we will provide quiz windows (available time slots) for students to register for their preferred time. Quizzes will feature randomized questions, so each student may receive a slightly different version of the quiz.
+
+During the quiz window, students are **NOT** allowed to disclose, discuss, or share any part of the quiz with any other individual, except as directly permitted or required by the course instructors. This includes discussions in person, online, or through any electronic means. Violation of this policy will result in academic penalties, which may include failure of the quiz or failure of the course.
+
+### Digital Cheatsheets
+
+Students are now allowed to use digital cheatsheets, which can be uploaded to the PrairieLearn system. This means students can prepare a digital double-sided A4 cheatsheet and submit it directly within the PrairieLearn interface before starting their quiz.
+
 ### PrairieTest
 
 You must complete the instruction below before writing quizzes. This is an essential step to check in the quiz room.
-- Open https://ca.prairietest.com/pt
+- Open [PrairieTest](https://ca.prairietest.com/pt)
 - Click Login and select University of British Columbia (ubc.ca)
 - Log in with your CWL
 - Accept the invitation to join MDS 5XX
   - You only need to do this once, and please DM the course coordinator if you have issues viewing the invitation.
 - You will see a list of future quizzes, including quiz times and locations, under Exam reservations.
```

Comment:
Change to:
- You will be able to book quizzes and view your booked quiz times and locations, under Exam reservations.